client/packages/electron/src/electron.ts

```diff
@@ -412,95 +438,87 @@ app.addListener(
   }
 );
 
-const fileMenu: MenuItemConstructorOptions = {
-  label: 'File',
-  submenu: [{ role: 'quit' }],
-};
+function configureMenus(
+  window: BrowserWindow,
+  translations: Record<keyof typeof defaultTranslations, string>
+) {
+  const t = (key: keyof typeof defaultTranslations) => translations[key] || key;
+
+  // add a context menu which shows when the user right clicks
+  window.webContents.on('context-menu', (_event, params) => {
+    // Electron _should_ localise based on the roles... alas, at least for mac os: https://github.com/electron/electron/issues/26231
+    const template: Electron.MenuItemConstructorOptions[] = [
+      { role: 'cut', label: t('cut') },
+      { role: 'copy', label: t('copy') },
+      { role: 'paste', label: t('paste') },
+      { role: 'selectAll', label: t('select-all') },
+      { type: 'separator' },
+      { role: 'reload', label: t('reload') },
+    ];
+    const menu = Menu.buildFromTemplate(template);
+    menu.popup({ window, x: params.x, y: params.y });
+  });
 
-// const editMenu: MenuItemConstructorOptions = {
```

Comment:
these have been commented out for 2 years, figured I'd take out... they're not complex to redo if we want them back in the future 😅

Reply:
I thought they were a neat idea.. but we don't want them 🙏